File: CS/EFCore/CustomEventsAndResources.Win/Controllers/CustomEventWithUserResourcesController.cs
using CustomEventsAndResources.Module.BusinessObjects;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Scheduler.Win;
using DevExpress.Persistent.Base.General;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CustomEventsAndResources.Win.Controllers {
    public class CustomEventWithUserResourcesController : ObjectViewController<ListView, CustomEventWithUserResources> {
        protected override void OnViewControlsCreated() {
            base.OnViewControlsCreated();
            ((SchedulerListEditor)View.Editor).ResourcesMappings.Id = "ID";
        }
    }
}

File: VB/EFCore/CustomEventsAndResources.Win/Controllers/CustomEventWithUserResourcesController.vb
Imports CustomEventsAndResources.Module.BusinessObjects
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Scheduler.Win

Namespace CustomEventsAndResources.Win.Controllers

    Public Class CustomEventWithUserResourcesController
        Inherits ObjectViewController(Of ListView, CustomEventsAndResources.Module.BusinessObjects.CustomEventWithUserResources)

        Protected Overrides Sub OnViewControlsCreated()
            MyBase.OnViewControlsCreated()
            CType(View.Editor, SchedulerListEditor).ResourcesMappings.Id = "ID"
        End Sub
    End Class
End Namespace
